blog/2023-04-04-deprecating_polymer.md

```diff
@@ -0,0 +1,22 @@
+---
+author: Bram Kragten
+authorURL: https://twitter.com/bramkragten
+authorTwitter: bramkragten
+title: Deprecating Polymer
+---
+
+The Home Assistant frontend used to use the [Polymer Library](https://polymer-library.polymer-project.org/3.0/docs/about_30) for her web components. Polymer is since been deprecated and superseded by [Lit](https://lit.dev).
```

Comment:
```suggestion
The Home Assistant frontend used to use the [Polymer Library](https://polymer-library.polymer-project.org/3.0/docs/about_30) for her web components. Polymer has since been deprecated and superseded by [Lit](https://lit.dev).
```